### ISSUE TITLE
Accessibility - Methodology Screen - Missing Headings

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#27]
Why are there no headings on the ‘Methodology’ screen?

**Comment by aquilent-dgemoets** [comment]
See discussion for #24.

**Comment by jfc3-dol** [comment]
I reviewed the ILAB 'Sweat and Toil' application on OPA's 'Galaxy Tab S' and an ILAB employees 'LG Tribute 2' and both did not read any headings.

From my understanding two of the 'More information' screens (About the App and Methodology) where HTML pages and had the headings in them.

I thought the OSHA 'Heat Index' application had similar screens and the headings with their proper level were read for HTML screens before. The other Android screens that had what equated to headings were read too.

**Comment by jfc3-dol** [comment]
Not sure why TalkBack does not read the headings in the code like iOS does for HTML views.

**Comment by jfc3-dol** [comment]
There is now a heading level one (h1) on the page but what you expect to be heading level two (h2) items are not. They are being read as links only which they could be both a link and a heading level two (h2).

**Comment by NJICF** [comment]
The links in methodology screen had been made into h2 heading.

**Comment by jfc3-dol** [comment]
@NJICF,

I think you meant what looked like heading level two (h2) links are now only heading level two (h2) items and the same text in the paragraphs are now a link.

Everything seems to be working correctly, so I'm closing the links.
